Misc/NEWS.d/next/Library/2024-11-01-04-21-26.gh-issue-125413.Z-jjZq.rst

```diff
@@ -0,0 +1,2 @@
+Speed up :meth:`pathlib.Path.copy` by making use of
```

Comment:
Maybe add the speed-up factor? Users could then say "wow that's really better!"

Reply:
It's quite sensitive to the workload unfortunately! If you're copying a small number of large files, then the time spent copying data dominates any improvement from using `scandir()`.

Also, `Path.copy()` is new in 3.14 so users don't have much of a baseline for how fast it is.

Reply:
I see, thanks for explaining it to me!